vars.tf

```diff
@@ -127,6 +127,31 @@ variable "zone" {
 
 
 
+# GKE related variables
+variable "gke_version" {
+  description = "Kubernetes engine version"
+  type        = string
+  default     = "1.24.2-gke.1900"
+}
+
+variable "gke_linux_pool_version" {
+  description = "GKE Linux node pool version"
+  type        = string
+  default     = "1.22.10-gke.600"
```

Comment:
Why the node pool version differ from GKE version?

Reply:
Goode question, added to Monday, holding PR.

Reply:
Task set to review